dali/base/dasds.cpp

```diff
@@ -84,6 +84,8 @@ static unsigned readWriteTimeout = 60000;
 #define DEFAULT_LCIDLE_RATE 1          // 1 write transactions per idle period. <= this rate is deemed idle (suitable for save)
 #define STORENOTSAVE_WARNING_PERIOD 72 // hours
 
+constexpr const char * temporaryBinaryFileSaveName = "dali_store_tmp.bin";
```

Comment:
would be good to move this to inside method that uses it,
also true of TMPSAVENAME (and good to change to constexpr const char * too)